Formula/msgpack-cpp.rb

```diff
@@ -0,0 +1,54 @@
+class MsgpackCpp < Formula
```

Comment:
Should we call this `msgpack-cxx`?

Reply:
I saw most of the formulae used `cpp` part of the names, also the head branch is also with `cpp_master`